content/en/docs/tasks/configmap-secret/managing-secret-using-kubectl.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Managing Secret using `kubectl`
+title: Managing Secret using kubectl
```

Comment:
markdown code block format can not be rendered in title

Reply:
Please try to use different pr for the modification of Chinese and English content.

Reply:
see https://github.com/kubernetes/website/pull/23986